samples/client/petstore/java/jersey2-java6/src/main/java/io/swagger/client/model/AdditionalPropertiesClass.java

```diff
@@ -59,7 +47,7 @@ public AdditionalPropertiesClass putMapPropertyItem(String key, String mapProper
    * Get mapProperty
    * @return mapProperty
   **/
-  @ApiModelProperty(example = "null", value = "")
+  @ApiModelProperty(value = "")
```

Comment:
@ePaul I remember Tony Tam filed a PR to fix it. If you need the number, I can dig it out for you.